### PR TITLE
Download Button & Modal

### PR DESCRIPTION
**Pull request by connorKeir**
**Description**

Created Download button & Modal and implemented base functionality

**Testing**

Comparing to figma design. Test files created

**Extra Attention**

Unsure if I have written tests properly.

### CODE REVIEW

**Comment by jonachehilton** [comment]
Your right, you haven't written the tests properly. I know that because the tests are not passing. Did you run the tests? `cd client` `npm run test`. Running it will tell you what is wrong

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "jsonwebtoken": "^8.5.1",
     "nodemon": "^2.0.12",
     "pg": "^8.7.1",
+    "react-modal": "^3.14.3",
```
Comment (jonachehilton) [inline package.json:35]: Remove react-modal

File: package.json [hunk 2]
```diff
@@ -43,6 +44,7 @@
     "eslint-plugin-import": "^2.23.4",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.24.0",
-    "eslint-plugin-react-hooks": "^4.2.0"
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "sass": "^1.42.1"
```
Comment (jonachehilton) [inline package.json:48]: Remove sass. This is the server package.json - We already have sass in the client/package.json

**Review by jonachehilton — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (connorKeir) [reply package.json:48]: I didn't see this in my version, wouldn't let me commit. Got it too commit should be good now.

**Pull request merged**